qa/suites/upgrade/jewel-x/parallel/3-upgrade-sequence/upgrade-mon-osd-mds.yaml

```diff
@@ -36,9 +36,3 @@ upgrade-sequence:
        daemons: [osd.2, osd.3]
        wait-for-healthy: false
        wait-for-osds-up: true
-   - exec:
```

Comment:
I couldn't figure it out, either, so I just dropped both commits and redid the cherry-pick.